modules/cudf/src/column/json.cpp

```diff
@@ -21,16 +21,16 @@ namespace nv {
 
 Column::wrapper_t Column::get_json_object(std::string const& json_path,
                                           rmm::mr::device_memory_resource* mr) {
+  auto options = cudf::strings::get_json_object_options{};
+  options.set_missing_fields_as_nulls(true);
```

Comment:
I don't think we want this? This'll set null fields to `'null'`.

Reply:
Or rather, we probably do want this but we want it to be an option the user can specify from JS.